### ISSUE TITLE
Search limited to filename

### ISSUE THREAD

**Issue by Luke-L** [yfszzx/stable-diffusion-webui-images-browser#83]
Shouldn't search cover the filename _and_ the prompts from the metadata? its very annoying to have to use a 3rd party program to search tags from metadata.

**Comment by AlUlkesh** [comment]
I suspect this wasn't done, because opening several 10000 files and reading that info would probably be rather slow. A database would be really useful here.

**Comment by Luke-L** [comment]
> I suspect this wasn't done, because opening several 10000 files and reading that info would probably be rather slow. A database would be really useful here.

lol you mean like the csv log file with all your gens and prompts? and i def dont have 10k files, so not sure what youre doing with those

**Comment by AlUlkesh** [comment]
No, the csv file is optional, default is to store that data inside the image files.

And  if you look at issue 80, someone has run into a 64K limit. So yes, people keep them.

**Comment by Luke-L** [comment]
ooookay, so just because someone hit the limit means it shouldnt be an option for the rest of us? seems like it shouldnt be that hard to implement
